src/Visualization/Python/GenerateXdmf.py

```diff
@@ -64,6 +64,8 @@ def generate_xdmf(file_prefix, output, subfile_name, start_time, stop_time,
         for h5file in h5files:
             h5temporal = h5file[0].get(subfile_name + '.vol').get(
                 id_and_value[0])
+            if not h5temporal:
```

Comment:
Do you think it would be worth adding a comment that the reason this is done is so that if there's missing data you can at least view what was dumped? Up to you :)